frontend/packages/ceph-storage-plugin/src/components/modals/storage-class-dropdown.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { Firehose } from '@console/internal/components/utils';
+import { StorageClassDropdownInner } from '@console/internal/components/utils/storage-class-dropdown';
+import { K8sResourceKind } from '@console/internal/module/k8s';
+
+const cephStorageProvisioners = ['ceph.rook.io/block', 'cephfs.csi.ceph.com', 'rbd.csi.ceph.com'];
+
+export const OCSStorageClassDropdown: React.FC<OCSStorageClassDropdownProps> = (props) => (
+  <Firehose resources={[{ kind: 'StorageClass', prop: 'StorageClass', isList: true }]}>
+    <StorageClassDropdown {...props} />
+  </Firehose>
+);
+
+const StorageClassDropdown = (props: any) => {
+  const scConfig = _.cloneDeep(props);
+  /* 'S' of Storage should be Capital as its defined key in resourses object */
+  const scLoaded = _.get(scConfig.resources.StorageClass, 'loaded');
+  const scData = _.get(scConfig.resources.StorageClass, 'data', []) as K8sResourceKind[];
+
+  const filteredSCData = scData.filter((sc: K8sResourceKind) => {
+    return cephStorageProvisioners.every(
+      (provisioner: string) => !_.get(sc, 'provisioner').includes(provisioner),
```

Comment:
Are you sure you want `includes`? That is a substring match.

Reply:
@spadgett right now provisioners are coming in the form of `NAMESPACE.cephfs.csi.ceph.com` `NAMESPACE.rbd.csi.ceph.com` `ceph.rook.io/block`, where Namespace is not constant and can be more than one word, so used `includes` to check to substring that will be present for sure.

Reply:
OK, I see. The storage stuff is not in my wheelhouse :)

Consider https://lodash.com/docs/4.17.15#endsWith